provisioning/ansible/roles/web/tasks/main.yml

```diff
@@ -40,8 +40,8 @@
 - name: Copy SSL configuration
   copy: src=ssl.conf dest=/etc/nginx/conf.d/ssl.conf owner=nginx group=nginx mode=600
 
-- name: Create PHP session directory
-  shell: mkdir /var/lib/php/session creates=/var/lib/php/session
+- name: Creates directory
+  file: path=/var/lib/php/session state=directory owner=nginx group=nginx mode=0775
```

Comment:
Did it create any problems not being owned by Nginx? Each separate application [gets its own session directory](https://github.com/SURFnet/Stepup-Deploy/blob/167b8bca3a9d003f6e4310650700264c81050be8/provisioning/ansible/roles/web/templates/php-fpm-pool.conf.j2#L228) owned by its own user.